package.json

```diff
@@ -40,6 +40,6 @@
     "es6-object-assign": "^1.1.0",
     "is-nan": "^1.2.1",
     "object-is": "^1.0.1",
-    "util": "^0.12.0"
+    "util": "^0.12.4"
```

Comment:
this is fine, but is 100% unnecessary because of the `^` range. You can update to anything within `0.12.x` with the range `^0.12.0`, and anyone freshly installing this package will get the latest version of `util` automatically.

Reply:
Ah, I see. I found that `next` has its 0.12.3 in the `package-lock.json` that's probably why.